### PR TITLE
Update console SDK to run on 4.10

### PR DESCRIPTION
**Pull request by rawagner**
Updates console SDK to make the plugin runnable on 4.10. But Im not entirely sure that the 0.0.5 version is a correct one.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,23 +22,24 @@
     "run-dev": "yarn build-dev && yarn http-server"
   },
   "dependencies": {
-    "@patternfly/react-core": "4.147.0",
-    "@patternfly/react-table": "4.61.14",
     "@types/node": "^14.14.34",
+    "@patternfly/react-core": "4.162.2",
+    "@patternfly/react-table": "4.31.7",
```
Comment (yzhao583) [inline package.json:27]: @rawagner Why do we need to change the version for `@patternfly/react-core` and `@patternfly/react-table`? is this required by the change of the `dynamic-plugin-sdk`?

File: package.json [hunk 2]
```diff
@@ -22,23 +22,24 @@
...
     "i18next": "^19.8.3",
     "lodash": "^4.17.21",
     "react": "^17.0.1",
     "react-dom": "^17.0.1",
     "react-helmet": "^6.1.0",
-    "react-i18next": "^11.8.11",
-    "react-router": "5.1.2",
-    "react-router-dom": "5.1.2",
+    "react-i18next": "^11.7.3",
+    "react-router": "5.2.0",
```
Comment (yzhao583) [inline package.json:36]: @rawagner Why do we need to change the version for `react-i18next` and `react-router` and `react-router-dom`? is this required by the change of the `dynamic-plugin-sdk`?

**Review by yzhao583 — CHANGES_REQUESTED** [review]

**Comment by yzhao583** [comment]
@rawagner In my local, I see this error, could you please help? Much appreciated!
image

**Review by tchughesiv — APPROVED** [review]
lgtm

File: package.json (hunk 1 above)
Comment (rawagner) [reply package.json:27]: @yzhao583 these are singleton modules defined in https://github.com/openshift/console/blob/release-4.10/frontend/packages/console-dynamic-plugin-sdk/src/shared-modules.ts#L4-L19 

its best to align with version defined by `dynamic-plugin-sdk` as they are the ones being used once the plugin is deployed

File: package.json (hunk 2 above)
Comment (rawagner) [reply package.json:36]: same reason as https://github.com/RHEcosystemAppEng/dbaas-dynamic-plugin/pull/89#discussion_r826405028

**Comment by rawagner** [comment]
> @rawagner In my local, I see this error, could you please help? Much appreciated! image

that looks like you didnt run `yarn install` ? Can you try again ?

**Pull request merged**